validator/project_validator.go

```diff
@@ -2346,6 +2346,16 @@ func checkBuildVariants(project *model.Project) ValidationErrors {
 				},
 			)
 		}
+
+		if len(buildVariant.EmptyTaskSelectors) > 0 {
+			errs = append(errs,
+				ValidationError{
+					Message: fmt.Sprintf("buildvariant '%s' contains empty task selectors: '%s'", buildVariant.Name, strings.Join(buildVariant.EmptyTaskSelectors, "', '")),
```

Comment:
nit -- this should also not use the words "empty task selectors"; this isn't going to be clear to users. Maybe something like "some buildvariant task names/tags don't match any tasks"